src/functions/advanced-functions/event.php

```diff
@@ -33,8 +33,10 @@
 	 * - EventShowMap bool - Set to true to embed the map in the Event view.
 	 * - EventCost string - Default cost of the Event.
 	 * - EventURL string - Link to the Event Website or Third-Party page
+ 	 * - FeaturedImage string - URL or ID of a featured image
```

Comment:
:no_entry_sign: Tabs must be used to indent lines; spaces are not allowed

<sub>hash: f3b103628f2247b71f57377a21e819ec</sub>